nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/provenance.css

```diff
@@ -178,31 +178,26 @@ div.flowfile-container {
     width: 282px;
     height: 120px;
```

Comment:
@scottyaslan I think we can remove the `height` property all together so the Children FlowFiles heading will not sit so far below Parent FlowFile heading